pkg/workloads/docker.go

```diff
@@ -359,7 +359,7 @@ func (d *dockerClient) getEndpointByIP(cont *dTypes.ContainerJSON) *endpoint.End
 		if contNetwork.GlobalIPv6Address != "" {
 			id := endpointid.NewID(endpointid.IPv6Prefix, contNetwork.GlobalIPv6Address)
 			if ep, err := endpointmanager.Lookup(id); err != nil {
-				log.WithError(err).Warningf("Unable to lookup endpoint by IP prefix %s", id)
+				log.WithError(err).WithField(logfields.EndpointID, id).Warning("Unable to lookup endpoint by IP prefix")
```

Comment:
It's a bit surprising here that we say "lookup by IP prefix" and then print the field of the endpoint ID. I guess it makes sense because the endpoint ID is derived from the IPv6 address, but it'd be less surprising if we also printed what that prefix is :)

Reply:
Changed field key to be V6Prefix and V4Prefix.